config/initializers/content.rb

```diff
@@ -0,0 +1,5 @@
+Workarea.configure do |config|
+  config.content_areas.merge!(
```

Comment:
(minor) A simpler way to do this would be `config.content_areas['customization'].append('autocomplete')`